src/whiteflag/wf_codec/encoding.rs

```diff
@@ -48,7 +48,6 @@ impl Encoding {
                 encode_bdx(remove_all_invalid_hex_characters(value))
             }
             EncodingKind::LAT | EncodingKind::LONG => encode_latlong(value),
-            _ => vec![0],
```

Comment:
@Romulus10 only reason I added this is because that is what was in the java codebase